src/main/java/org/distributed/systems/chord/messaging/JoinMessage.java

```diff
@@ -4,59 +4,14 @@
 
 public class JoinMessage {
 
-        public static class JoinRequest implements Command {
+    public static class JoinRequest implements Command {
 
-            public long requestorKey;
-            public ActorRef requestor;
+        public long requestorKey;
```

Comment:
public final long, would be nice for immutability

Reply:
:heavy_check_mark: